recipes/cyflann/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "cyflann" %}
+{% set version = "0.2.3" %}
+{% set sha256 = "a38cfdbaf7b4005f1f6ba417186f485483f78219bec5aa95f479f38551d234ae" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip:
+    # Windows should work, but flann recipe isn't built there yet.
+    - True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - numpy x.x
+    - flann >=1.9.1,<1.10
+    - cython
+  run:
+    - python
+    - numpy x.x
+    - flann >=1.9.1,<1.10
```

Comment:
`curl` has been really stable. So haven't worried about it, but could be worth revisiting.

Sure it is worth doing these as well. May want to break up the PRs, but it is up to you.